### ISSUE TITLE
Post in english, please!

### ISSUE THREAD

**Issue by rogeriodec** [Stephan-S/FS19_AutoDrive#661]
Although apparently the developers of this mod are German, as you can see, it is being used by many languages.
And English is the default language in this case.
Would it be possible to show a guide for users to post issues **ONLY in English**?
This makes possible not only greater intelligibility, but especially when doing a Google search, it is usually done in English, so it would be easier to find the answer, unlike if the issue is written in another language.

**Comment by Stephan-S** [comment]
@rogeriodec 
There is only one developer here ;-)

Well, not everyone is fluent in english and I prefer to receive issues that actually communicate the issue instead of focusing on language barriers. So while I prefer issues to be opened and handled in english, I will not force it on everyone and I will of course still respond to issues written in german. If you are interested, you can always use google transalate or DeepL to follow them or if the headline sounds interesting, ask directly for a translation there.

Like I said, I can't or wont't force any language here. But I prefer english :-)

**Comment by rogeriodec** [comment]
Ok, it's your decision since the mod is yours, I just suggested something that would make it more internationally accessible.

**Comment by Stephan-S** [comment]
Like I said. I also prefer english but not everyone is fluent enough

**Comment by raxtor** [comment]
Maybe just rule "Thread title in english" would be nice.. than we all know which issue is reported..
@rogeriodec anyway you can use translation option for quick oriantation

**Comment by rogeriodec** [comment]
I know I can use a translator, but mainly who is creating the issue might use it first!
Google translator is being very efficient these days...
Also, the German language has many similarities to English.
Also, German culture is a lot more improved than, for example, my country's culture...
Therefore, I do not think that there is any great difficulty for a German to write in English.
And in the end, the creator of the issue will do the translation work only once, while on the contrary, if 1000 people from other countries see the issue, 1000 people will have to do the translation.
Also, as I mentioned earlier, if I search for an Autodrive issue in Google, I will usually do it in English, so if the article is written in German, the solution will not be found, thus I will create a new issue, generating more work for you to answer, something that would be unnecessary if the original issue was already in English ...
